oqamaint/opensuse.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+from update import Update
+
+
+class openSUSEUpdate(Update):
+
+    @staticmethod
+    def repo_prefix():
+        return 'http://download.opensuse.org/repositories'
+
+    @staticmethod
+    def maintenance_project():
+        return 'openSUSE:Maintenance'
```

Comment:
Any reason not to use static properties instead which can obviously have a getter defined if more complex logic is needed as done in `stagingapi.py`?